egs/avspeech/local/loader/constants/__init__.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+STORAGE_DIR = os.environ.get("STORAGE_DIR", "storage_dir")
+
+AUDIO_MIX_COMMAND_PREFIX = "ffmpeg -y -t 00:00:03 -ac 1 "
+
+AUDIO_DIR = f"../../{STORAGE_DIR}/storage/audio"
+VIDEO_DIR = f"../../{STORAGE_DIR}/storage/video"
+EMBED_DIR = f"../../{STORAGE_DIR}/storage/embed"
+MIXED_AUDIO_DIR = f"../../{STORAGE_DIR}/storage/mixed"
+SPEC_DIR = f"../../{STORAGE_DIR}/storage/spec"
```

Comment:
Won't this have weird behavior when `STORAGE_DIR` is a absolute path?

Reply:
Okay, I will make that as an absolute path instead of keeping it inside `avspeech/`. Should I document that it should be an absolute path?
Like, 
`cd path/to/storage`
`export STORAGE_DIR=$(pwd)`

Reply:
Maybe, in constants, only have relative path (with respect to the storage directory that's is provided in command line)

Reply:
```suggestion

STORAGE_DIR = os.environ.get("STORAGE_DIR", "../../storage_dir")
AUDIO_DIR = f"{STORAGE_DIR}/storage/audio"
VIDEO_DIR = f"{STORAGE_DIR}/storage/video"
EMBED_DIR = f"{STORAGE_DIR}/storage/embed"
MIXED_AUDIO_DIR = f"{STORAGE_DIR}/storage/mixed"
SPEC_DIR = f"{STORAGE_DIR}/storage/spec"
```
Like this?

Reply:
That would do I guess !